File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/core/model/BasicInfo.kt
package com.syncrhrohealth.bandpass_ble_connection.core.model

data class BasicInfo(
    val isTempSensorAttached: Int,
    val deviceOperationMode: Int,
    val modeThermostatState: Int,
    val modeManualState: Int,
    val fuelLevel: Int,
    //
    val currentLevel: Int,
    val expectedLevel: Int,
    val currentTemp: Float,
    val expectedTemp: Float,
)

File: ios/Classes/Core/Model/BasicInfo.swift
//
//  BasicInfo.swift
//  Runner
//
//  Created by MAC on 10/3/25.
//

import Foundation

struct BasicInfo {
    let isTempSensorAttached: Int
    let deviceOperationMode: Int
    let modeThermostatState: Int
    let modeManualState: Int
    let runState: Int
    let fuelLevel: Int
    
    let currentLevel: Int
    let expectedLevel: Int
    let currentTemp: Float
    let expectedTemp: Float
}
